app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -17,6 +18,44 @@
   color: govuk-colour("white");
 }
 
-.gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:before {
+.gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:after {
   border-color: govuk-colour("white");
 }
+
+.govuk-breadcrumbs .govuk-breadcrumbs__list-item {
+  margin-left: 0;
+  padding-left: 0;
+  margin-right: 20px;
+  @include chevron($direction: "right", $color: govuk-colour("dark-grey"));
+
+  &:before,
+  &:last-of-type:after {
+    display: none;
+  }
+
+  @include govuk-media-query($until: tablet) {
+    display: none;
+    margin-bottom: 0;
+    padding: 14px 0;
+
+    &:last-of-type {
+      display: inline-block;
+      padding-left: 0;
+      margin-right: 0;
+      max-width: 76%;
```

Comment:
This still seems to be assuming that the first item in the breadcrumb is no more than the remaining width because it only contains the word 'Home'.

This isn't the case, as shown in the component guide itself (and on [pages like this](https://www.gov.uk/service-manual/helping-people-to-use-your-service)). Example below (I've artificially extended the length of the last item's text, but something like this could happen).

<img width="601" alt="Screenshot 2020-02-07 at 08 25 37" src="https://user-images.githubusercontent.com/861310/74013005-70c93780-4983-11ea-9fa4-b2267b00121a.png">

If you're going to set the width of the last item, you'll have to set the width of the first item as well.

Reply:
added max-width to the first of type as well as the last